src/tracer/MOM_lateral_boundary_diffusion.F90

```diff
@@ -584,13 +582,10 @@ subroutine fluxes_layer_method(boundary, ke, hbl_L, hbl_R, h_L, h_R, phi_L, phi_
   real, allocatable :: F_layer_z(:)  !< Diffusive flux at U/V-point in the ztop grid    [H L2 conc ~> m3 conc]
   real              :: h_vel(ke)     !< Thicknesses at u- and v-points in the native grid
                                      !! The harmonic mean is used to avoid zero values      [H ~> m or kg m-2]
-  real    :: khtr_avg                !< Thickness-weighted diffusivity at the velocity-point [L2 T-1 ~> m2 s-1]
-                                     !! This is just to remind developers that khtr_avg should be
-                                     !! computed once khtr is 3D.
   real    :: htot                    !< Total column thickness                              [H ~> m or kg m-2]
-  integer :: k, k_bot_min, k_top_max !< k-indices, min and max for bottom and top, respectively
-  integer :: k_bot_max, k_top_min    !< k-indices, max and min for bottom and top, respectively
-  integer :: k_bot_diff, k_top_diff  !< different between left and right k-indices for bottom and top, respectively
+  integer :: k, k_bot_min            !< k-indices, min and max for bottom and top, respectively
```

Comment:
k_top_max, k_top_min, and k_top_diff are in commented out code, and are reasonably described in the comments.  Even though they are not used yet, perhaps they should be retained in comments.